client.go

```diff
@@ -693,6 +693,20 @@ func authorizeHandler(fn func(*Client, url.Values, *CSRF) error) AuthorizeOption
 	return &authorizeOptionFn{fn: fn}
 }
 
+func WithResponseType(responseType string) AuthorizeOption {
```

Comment:
let's make it `WithResponseType(responseTypes ...string)` as they might be multiple response types